lib/template.js

```diff
@@ -72,12 +72,18 @@ template.read = function(templatePath, options, callback) {
  * @param {object} templateBody - a parsed CloudFormation template
  * @returns {array} a set of questions for user prompting
  */
-template.questions = function(templateBody, overrides) {
+template.questions = function(templateBody, overrides, options) {
   overrides = overrides || {};
   overrides.defaults = overrides.defaults || {};
   overrides.messages = overrides.messages || {};
   overrides.choices = overrides.choices || {};
 
+  options = options || {};
+  options.id = options.id || 'alias/cloudformation';
+
```

Comment:
Could you please test the alternative code paths here? ie where options is not provided and where options.id is provided?